### PR TITLE
Order: Change ID for all operations to spSubscriptionId #77

### PR DESCRIPTION
**Pull request by junetaw**
Order: Change ID for all operations to spSubscriptionId #77

### CODE REVIEW

File: 2.6.0/spec/orders.md [hunk 1]
```diff
@@ -644,7 +650,7 @@ The date and time when the order is expected to be delivered. This is an estimat
 ### spSubscriptionId
 
 * Data format: [spSubscriptionId](../common/dataformats.md#spsubscriptionid)
-* Required in response, if set
+* Required
 
 ### subscriptionId
```
Comment (edruid) [inline 2.6.0/spec/orders.md:655]: The line with `Required for the operations SUSPEND, RESUME, MODIFY and CHANGE` below is no longer needed.

File: 2.6.0/spec/orders.md [hunk 2]
```diff
@@ -644,7 +650,7 @@ The date and time when the order is expected to be delivered. This is an estimat
 ### spSubscriptionId
 
 * Data format: [spSubscriptionId](../common/dataformats.md#spsubscriptionid)
-* Required in response, if set
+* Required
```
Comment (edruid) [inline 2.6.0/spec/orders.md:653]: Don't we in general use "Mandatory" rather than "Required"?

File: 2.6.0/spec/orders.md (hunk 1 above)
Comment (edruid) [inline 2.6.0/spec/orders.md:655]: This is probably out of scope, but could we rename `subscriptionId` to `coSubscriptionId`? So it's easier to understand what is what.

Comment (junetaw) [reply 2.6.0/spec/orders.md:655]: Yes, that would be very good. Everything that avoids mistakes is good.

**Comment by teemufrosen** [comment]
Working group meeting: Needs further discussion, moving to Will not do for 2.6.

**Pull request closed without merging**